### PR TITLE
MAINT - Moved Schema validation from Tests to CoreVerifier and removed the sp module

### PR DESCRIPTION
**Pull request by blen-desta**
#### What does this PR do (if it's not clear from the Title)? Please include any specification references that apply.

- Moved Schema validation from Tests to CoreVerifier
- Removed the sp module
- Updated docs

#### Checklist:
- [ ] SAML Spec Table of Contents documentation updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,12 +89,13 @@ This module will contain all tests being written against a SAML IdP. The `src` d
 
 This [coverage](ctk/idp/coverage) directory is used to track which sections of each SAML specification are covered by these tests:
 
-#### sp
-This module will contain all tests being written against a SAML SP. The src directory of the module is organized identically to the idp module.
-
```
Comment (brjeter) [inline README.md]: It feels so good to have all this orphaned SP stuff removed. So much cleaner!

Comment (blen-desta) [reply README.md]: ☝️

**Comment by blen-desta** [comment]
build now

**Comment by blen-desta** [comment]
rebased
build now

**Pull request merged**